matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineSettings.kt

```diff
@@ -32,6 +32,10 @@ data class TimelineSettings(
          * A flag to filter redacted events
          */
         val filterRedacted: Boolean = false,
+        /**
+         * A flag to filter useless events, such as membership events without any change
+         */
+        val filterUseless: Boolean = false,
```

Comment:
isUseless seems a bit hard for a sdk naming

Reply:
It's true anyway, it's useless...

Reply:
Yes but it sounds a bit weird to use as a sdk user... but ok for me anyway